wp-content/themes/missilethreat/assets/_scss/components/_tables.scss

```diff
@@ -8,6 +8,10 @@
     border-collapse: collapse;
   }
 
+  .wp-block-table {
+    overflow-x: hidden;
```

Comment:
So I know this was added because of horizontal scroll on tables, but on mobile I think we would want to keep this, right? Otherwise a wide table wouldn't be accessible.